
From: Kumar Gala <kumar.gala@freescale.com>

Added myself to the MAINTAINERS file for 85xx.  Added an entry into the
CREDITS file for me.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/CREDITS     |    6 ++++++
 25-akpm/MAINTAINERS |    7 +++++++
 2 files changed, 13 insertions(+)

diff -puN CREDITS~add-ppc85xx-maintainers-entry CREDITS
--- 25/CREDITS~add-ppc85xx-maintainers-entry	2004-06-22 12:48:03.018466584 -0700
+++ 25-akpm/CREDITS	2004-06-22 12:48:03.025465520 -0700
@@ -1060,6 +1060,12 @@ S: USA
 N: Jeff Garzik
 E: jgarzik@pobox.com
 
+N: Kumar Gala
+E: kumar.gala@freescale.com
+D: Embedded PowerPC 6xx/7xx/74xx/82xx/85xx support
+S: Austin, Texas 78729
+S: USA
+
 N: Jacques Gelinas
 E: jacques@solucorp.qc.ca
 D: Author of the Umsdos file system
diff -puN MAINTAINERS~add-ppc85xx-maintainers-entry MAINTAINERS
--- 25/MAINTAINERS~add-ppc85xx-maintainers-entry	2004-06-22 12:48:03.019466432 -0700
+++ 25-akpm/MAINTAINERS	2004-06-22 12:48:03.027465216 -0700
@@ -1284,6 +1284,13 @@ W:	http://www.penguinppc.org/
 L:	linuxppc-embedded@lists.linuxppc.org
 S:	Maintained
 
+LINUX FOR POWERPC EMBEDDED PPC85XX
+P:     Kumar Gala
+M:     kumar.gala@freescale.com
+W:     http://www.penguinppc.org/
+L:     linuxppc-embedded@lists.linuxppc.org
+S:     Maintained
+
 LLC (802.2)
 P:	Arnaldo Carvalho de Melo
 M:	acme@conectiva.com.br
_
